core/pkg/opencost/bingen.go

```diff
@@ -72,4 +70,4 @@ package opencost
 // @bingen:generate:CloudCostLabels
 // @bingen:end
 
-//go:generate bingen -package=opencost -version=17 -buffer=github.com/opencost/opencost/core/pkg/util
+//go:generate bingen -package=opencost -version=18 -buffer=github.com/opencost/opencost/core/pkg/util
```

Comment:
This part, I am a bit unsure about... Are these types on fields of on any other types? 

```
// @bingen:generate:Coverage
// @bingen:generate:CoverageSet
```

Reply:
Looks like no. I _think_ this is fine. Deletion is a tricky though.